### PR TITLE
Converted to Maven project, added setup script, other small fixes

### PR DESCRIPTION
**Pull request by ekuiter**
To make this project more usable for non-Eclipse users, I added a POM for Maven and a run script, so it is only necessary to add a suitable options.ini. Now I only have to run ~/MibTeX/MibTeX.sh and everything works :)

Also I removed all JARs and replaced them with the latest Maven dependencies, which seems to work fine.

### CODE REVIEW

**Comment by pmbittner** [comment]
Wow great idea, thank you! I will try to look at the changes next week.

File: MibTeX/resources/index_in.html [hunk 1]
```diff
@@ -2,7 +2,7 @@
 <html lang="en">
     <head>
         <meta charset="UTF-8">
-        <title>MibTex</title>
+        <title>MibTeX</title>
```
Comment (pmbittner) [inline MibTeX/resources/index_in.html:5]: good catch!

**Review by pmbittner — APPROVED** [review]
Changes look good! 👍  I think the `MibTeX.sh` is helpful, but in general MibTeX is a multi-purpose tool and there is no single default way to run it. I usually run MibTeX with different ini files for different use-cases (mostly NEW_HTML and TYPO3). Yet, as it is basically a helper script anyway I am fine with having it.

**Comment by pmbittner** [comment]
@tthuem Should me merge? I am fine with that but it might disrupt your workflow.

**Comment by tthuem** [comment]
Let's give it a try.

**Pull request merged**